helm/bundles/cortex-manila/values.yaml

```diff
@@ -0,0 +1,179 @@
+# Copyright 2025 SAP SE
+# SPDX-License-Identifier: Apache-2.0
+
+owner-info:
+  helm-chart-url: "https://github.com/cobaltcore-dev/cortex/helm/bundles/cortex-manila"
+  maintainers:
+    - "p.matthes@sap.com"
+    - "markus.wieland@sap.com"
+    - "arno.uhlig@sap.com"
+  support-group: "workload-management"
+  enabled: true
+
+alerts:
+  prometheus: openstack
+
+# Custom configuration for the cortex postgres chart.
+cortex-postgres:
+  serviceMonitor:
+    extraLabels:
+      # The name of the prometheus instance to use for scraping metrics.
+      prometheus: openstack
+    matchLabels: # See below, this is used to select the postgresql service.
+      app: cortex-manila-postgresql
+  postgresql:
+    fullnameOverride: cortex-manila-postgresql
+    primary:
+      service:
+        labels: # See above, this is used to select by the service monitor.
+          app: cortex-manila-postgresql
+      # Overrride the names of the extra volumes to match the manila chart.
+      extraVolumes:
+        - {name: pre-upgrade-script, configMap: {name: cortex-manila-postgresql-pre-upgrade-script}}
+        - {name: post-upgrade-script, configMap: {name: cortex-manila-postgresql-post-upgrade-script}}
+
+# Custom configuration for the cortex mqtt chart.
+cortex-mqtt:
+  rabbitmq:
+    fullnameOverride: cortex-manila-mqtt
+
+# Custom configuration for the cortex core chart.
+cortex-core:
+  fullnameOverride: cortex-manila
+  service:
+    extraLabels:
+      app: cortex-manila # See below, used for the service monitor.
+  serviceMonitor:
+    matchLabels: # See above, this is used to select the postgresql service.
+      app: cortex-manila
+  # Modes in which the service should be run.
+  # Each mode is translated into a separate service and deployment.
+  modes:
+    # The name of the mode will be used for the kubernetes deployments.
+    # The args will be passed to the service as command line arguments, meaning
+    # the passed arg need to be baked into the service binary.
+    #
+    # Syncer syncs data from external sources into the database(s).
+    - name: syncer
+      args: ["syncer"]
+      replicas: 1
+
+    # Extractor extracts features from the synced data.
+    - name: extractor
+      args: ["extractor"]
+      replicas: 1
+
+    # Service that extracts and exposes KPIs.
+    - name: kpis
+      args: ["kpis"]
+      replicas: 1
+
+    # External scheduler for Manila.
+    - name: scheduler
+      args: ["scheduler-manila"]
+      replicas: 1
+
+  # Values passed to the service through a configmap.
+  conf:
```

Comment:
The config is rather complex and contains nested structures.
Would it be possible to move the scheduling config to dedicated files?

The proposed structure would look like:
```
.
└── cortex
    ├── Chart.yaml
    ├── etc
    │       ├── nova.yaml # Template free, plain-yaml config incl. validation during PR creation
    ├── templates
    │       ├── include etc/nova.yaml
```

Reply:
Future topic in my opinion. Yes, the config will likely be split into sub-configs anyways. But we need a clear plan for how to do that since it also impacts how we configure the local development setup and other things. Therefore I would say, we should revisit this in a separate PR. This is not related to the deployment separation.